molo/surveys/templates/surveys/molo_survey_page.html

```diff
@@ -25,7 +25,7 @@ <h4 class="surveys__intro">{{ page.intro }}</h4>
             {% endif %}
           </fieldset>
         {% endfor %}
-        <input type="submit" value="{% if is_intermediate_step %}{% trans 'Next Question' %}{% else %}{% trans 'Submit Survey' %}{% endif %}" />
+        <input type="submit" value="{% if is_intermediate_step %}{% trans 'Next Question' %}{% else %}{{self.submit_text}}{% endif %}" />
```

Comment:
@codiebeulaine let's add the exciting text as a default value here, like this 
{% trans "Submit Survey" as text %}
`{{self.submit_text|default:text }}`